anime/Services/ImportTeamService.php

```diff
@@ -23,6 +23,8 @@
 //     'source'       Absolute URL to the published CSV of the Google Spreadsheet meeting the format
 //                    restrictions mentioned below.
 //
+//     'password_salt'  Random string to be used as password salt.
```

Comment:
nit: it's a Fowler-Noll-Vo hash, but I guess it can be considered random from the consumer's point of view.